salt/orch/apply-configurations.sls

```diff
@@ -0,0 +1,16 @@
+# update registries
+apply-docker-config:
+  salt.state:
+    - tgt: 'roles:kube-(master|minion)'
+    - tgt_type: grain_pcre
+    - sls:
+      - docker
+
+reload-docker-daemon:
+  salt.state:
+    - tgt: 'roles:kube-(master|minion)'
+    - tgt_type: grain_pcre
+    - sls:
+      - docker.reload
+    - require:
+      - salt: apply-docker-config
```

Comment:
The whole file could be reduced to:

```SaltStack
# update registries
apply-docker-config:
  salt.state:
    - tgt: 'roles:kube-(master|minion)'
    - tgt_type: grain_pcre
    - sls:
      - docker
      - docker.reload
```

And that makes me wonder if we really need a new orchestration for this. Maybe we could do the `docker.reload` with some call to the Salt API...

Reply:
thx for the hint. looking at it again i also believe we should include the admin in the target...